plugins/services/src/js/structs/ServiceTree.js

```diff
@@ -110,6 +110,10 @@ module.exports = class ServiceTree extends Tree {
     return this.enforceRole;
   }
 
+  getNoLimit() {
```

Comment:
We need to add this function to the type definition in `ServiceTree.d.ts`

Reply:
Good point.

Reply:
Done.